test/DecryptionFailureTracker-test.ts

```diff
@@ -203,61 +208,61 @@ describe('DecryptionFailureTracker', function() {
 
         tracker.trackFailures();
 
-        expect(count).toBe(1, 'should only track a single failure per event');
+        // should only track a single failure per event
+        expect(trackEventSpy).toHaveBeenCalledTimes(1);
 
         done();
     });
 
-    xit('should not track a failure for an event that was tracked in a previous session', (done) => {
-        // This test uses localStorage, clear it beforehand
-        localStorage.clear();
+    // this functionality is commented out in DecryptionFailureTracker
+    // commenting as well as xit here to avoid fixing ts issues for dead code
```

Comment:
can we just delete it? (and the code in DecryptionFailureTracker)

we have git history if it needs to be recovered.

Reply:
Reverted in favour of #8272